IFC4x3/Properties/a/ApparentPower_1n4$pozW90Wu_Sinh6vpNI/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="ApparentPower_1n4pozW90Wu_Sinh6vpNI" Name="ApparentPower" UniqueId="7113fcf2-f602-4083-8f9c-b31ac6e735d2" PrimaryDataType="IfcPowerMeasure" />
```

Comment:
Can you clarify the difference between NominalPower and ApparentPower as the description on nominal power says: Apparent Power in it.

Reply:
Second this.

Reply:
Concept are realy close, so I have erased apparent power.